### PR TITLE
Func init added

### PR DESCRIPTION
**Pull request by JuanC1303**
I added the func init.

This function start the conecction with the mongo data base and try to connect to the collections that we will use. If the collections does not exist the program will create the collection.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"gopkg.in/mgo.v2/bson"
 
 	"github.com/rs/zerolog/log"
 	"google.golang.org/grpc"
 )
 
 const port string = "localhost:8080"
+const mongoURI string = "mongodb+srv://admin:admin@pl-health-service.s25udti.mongodb.net/test"
```
Comment (Merteg) [inline main.go]: Note: it's gonna work even without type specification

File: main.go [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
+	err = client.Connect(ctx)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	err = client.Ping(ctx, nil)
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+
+	targetCollection, err := client.Database("pl-health-service").ListCollectionNames(context.TODO(), bson.M{"name": "target"})
```
Comment (Merteg) [inline main.go]: you defined ctx on line 26. You should use it instead of context.TODO

File: main.go [hunk 3]
```diff
@@ -1,16 +1,59 @@
...
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	if len(targetCollection) == 0 {
+		_ = client.Database("pl-health-service").CreateCollection(context.TODO(), "target")
```
Comment (Merteg) [inline main.go]: Why do you ignore the error?

File: main.go (hunk 2 above)
Comment (Merteg) [inline main.go]: Create a constant for database name pls
Also for collection name
Note that collection names should be 'targets' and 'health'

File: main.go [hunk 4]
```diff
@@ -1,16 +1,59 @@
...
+		log.Info().Msg("target collection created and ready")
+	} else {
+		log.Info().Msg("target collection ready")
```
Comment (Merteg) [inline main.go]: Better to say "exist" not "ready"

File: main.go [hunk 5]
```diff
@@ -1,16 +1,59 @@
...
+	}
+	healthCollection, err := client.Database("pl-health-service").ListCollectionNames(context.TODO(), bson.M{"name": "health"})
+	if err != nil {
+		log.Fatal().Err(err)
+	}
+	if len(healthCollection) == 0 {
+		_ = client.Database("pl-health-service").CreateCollection(context.TODO(), "health")
```
Comment (Merteg) [inline main.go]: same here. Handle error, don't ignore it

File: main.go [hunk 6]
```diff
@@ -1,16 +1,68 @@
...
 	"github.com/rs/zerolog/log"
 	"google.golang.org/grpc"
 )
 
-const port string = "localhost:8080"
+const port = "localhost:8080"
+const mongoURI = "mongodb+srv://admin:admin@pl-health-service.s25udti.mongodb.net/test"
+const healthdb = "pl-health-service"
+const coll1 = "targets"
+const coll2 = "health"
```
Comment (Merteg) [inline main.go]:
```suggestion
const (
    port = "localhost:8080"
    mongoURI = "mongodb+srv://admin:admin@pl-health-service.s25udti.mongodb.net/test"
    dbName = "pl-health-service"
    targetsCollName = "targets"
    healthCollName = "health"
)
```

**Review by Merteg — CHANGES_REQUESTED** [review]
coll1 and coll2 is not cool)
Also you can wrap list of constants under single const ()

**Review by Merteg — APPROVED** [review]

**Review by crismxiex — APPROVED** [review]

**Pull request merged**